Java/String/JavaAnagrams.java

```diff
@@ -0,0 +1,53 @@
+// https://www.hackerrank.com/challenges/java-anagrams/problem
+
+import java.util.*;
+
+public class Solution {
+    static boolean isAnagram(String a, String b) { 
+        char array1[] = a.toLowerCase().toCharArray();
+        char array2[] = b.toLowerCase().toCharArray();
+        
+        sortArray(array1);
+        sortArray(array2);
+
+        if (array1.length != array2.length) {
+            return false;
+        } else {
+            int check = 0;
+
+            for (int i = 0; i < array1.length; i++) {
+                if (array1[i] != array2[i]) {
+                    check++;
+                }
+            }
+            
+            if (check != 0) {
+                return false;
+            } else {
+                return true;
+            }  
+        } 
+    }
+
+    public static void sortArray(char[] array) {
+        char tmp;
+        for (int i = 0; i < array.length; i++) {
+            for (int j = 0; j < array.length - 1; j++) {
+                if ((int)array[j] > (int)array[j + 1]) {

```

Comment:
Does it work without casting to `int`?

    if (array[j] > array[j + 1])

Reply:
no